bin/docker_run

```diff
@@ -1,5 +1,17 @@
 #!/bin/bash
 
+# Remove default old version of cmdtest/yarn
+apt uninstall cmdtest
+
+# Install new version of yarn
+curl -sS https://dl.yarnpkg.com/debian/pubkey.gpg | apt-key add -
+echo "deb https://dl.yarnpkg.com/debian/ stable main" | tee /etc/apt/sources.list.d/yarn.list
+apt update
+apt install -y yarn
```

Comment:
@KaustubhDamania  Shouldnt we set up the yarn in docker

Reply:
Do you mean yarn installation commands should be inside Dockerfile? In my case, the db: create and the commands after that weren't getting executed due to the yarn error so I decided to install it right before those commands get executed

Reply:
yes.https://hackernoon.com/using-yarn-with-docker-c116ad289d56

Reply:
@tesssie please check my latest commit, the yarn commands are in Dockerfile now